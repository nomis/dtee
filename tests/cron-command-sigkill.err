dtee: dtee@test/cron-command-sigkill.run: process terminated by signal 9: Killed
