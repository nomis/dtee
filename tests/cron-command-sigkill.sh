#!/bin/bash
. "$(dirname "$0")"/util/common.sh

run_test "-q" ./test-kill-pid 9
RET=$?

eval $(./test-waitpid ./dtee ./dtee "-q" ./test-kill-pid 9)
RET2=$?

variables_must_eq RET $((128 + 9)) \
	RET2 0 \
	WIFSIGNALED 0
