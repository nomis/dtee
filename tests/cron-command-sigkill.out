passthrough-stdout-stderr1.com